packages/e2e-tests/src/specs/dashboard/publisherLogo.js

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+
+/**
+ * External dependencies
+ */
+import {
+  deleteMedia,
+  uploadPublisherLogo,
+  visitSettings,
+  withExperimentalFeatures,
+} from '@web-stories-wp/e2e-test-utils';
+
+const ERROR_TEXT =
+  'Sorry, this file type is not supported. Only jpg, png, and static gifs are supported for publisher logos.';
+
+describe('publisher logo', () => {
+  withExperimentalFeatures(['enableSVG']);
+
+  it('should not upload a logo that is an invalid type with svg enabled', async () => {
+    await visitSettings();
```

Comment:
Should this be a `beforeEach`?

Reply:
Unnecessary optimization IMO, especially since `visitSettings` uses `expect()`, which is discouraged to do in `beforeEach` (and causes lint errors if used directly)